pkg/scene/widget_section_test.go

```diff
@@ -0,0 +1,76 @@
+package scene
+
+import (
+	"testing"
+
+	"github.com/reearth/reearth-backend/pkg/id"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestWidgetSection_Find(t *testing.T) {
+	wid := id.NewWidgetID()
+
+	was := NewWidgetAlignSystem()
```

Comment:
```suggestion
	section := NewWidgetSection()
```

WidgetAlignSystem cannot be used in `TestWidgetSection_*`